### ISSUE TITLE
QUESTION: Compatibilty with 3.4.X

### ISSUE THREAD

**Issue by raphaelwdrf** [konti-kun/redmine_mapping_board#35]
Hello,

Is this plugin compatible with 3.4.X?

**Comment by konti-kun** [comment]
Hi,

Have you tried?
I haven't written this app for a long time so I don't catch the latest version.

**Comment by raphaelwdrf** [comment]
Yes, i've tried it, and i got some unexpected things like cant choose background image, some errors, but i have a bunch of plugins and modifications.
I know you havent updated this plugin for a long time, so are you willing to update, then i'll take some time to test on a clean container and report errors as possible.
I work on non-agile projects, and i'm trying to include agile tools, i'm trying to use taiga.io and leantime software, but they are too much agile, redmine is good but gantt is outdated for me and we have no board to work on.
